constants.py

```diff
@@ -5,7 +5,7 @@
 
 # ML constants
 DEFAULT_BATCH_SIZE = 256
-MAX_BATCH_SIZE = 2**20
+MAX_BATCH_SIZE = 2 ** 20
```

Comment:
Really that much?! It seems it is anterior to this PR but I raise the question here: why? cc @RomainGoussault @aygalic

Reply:
I guess for very big datasets we need to make sure the batch size is not greater than this `MAX_BATCH_SIZE`. Actually this makes me think that sometimes we clip the batch size so that is not greater than MAX_BATCH_SIZE but we don't give any feedback to the user.

Reply:
I created an issue here https://github.com/SubstraFoundation/distributed-learning-contributivity/issues/232